app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CommentsController < ApplicationController
+  before_action :find_article, only: %i[create]
+  before_action :find_comment, only: %i[destroy]
+
+  http_basic_authenticate_with name: 'user', password: 'secret', only: :destroy
+
+  def create
+    @comment = @article.comments.create(comments_params)
```

Comment:
Is it necessary to create the `comment` fusing the `article`?

Reply:
[create comment](https://github.com/SanGeeky/rails-training/pull/1#discussion_r611967401)